tsconfig.json

```diff
@@ -22,6 +22,6 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true
   },
-  "include": ["src"],
+  "include": ["src", "src/globals.d.ts"],
```

Comment:
this makes `SmartWeave` and `ContractError` available across the project without having to redeclare them